website/templates/emails/reviews_submission_status.html.mako

```diff
@@ -29,6 +29,11 @@
     % endif
     </p>
 
+    % if is_admin:
```

Comment:
Should this also check for if the provider is PsyArXiv? They're the only ones who have chronos right now, and the help article is PsyArXiv specific.